src/main/resources/data.sql

```diff
@@ -0,0 +1,20 @@
+INSERT INTO notice(name, salary, image, job_position, content) VALUES('bossdog', 5000, 'https://cdn.vuetifyjs.com/images/cards/house.jpg', 'BACKEND','hi!');
+INSERT INTO notice(name, salary, image, job_position, content) VALUES('bossdog1', 5000, 'https://cdn.vuetifyjs.com/images/cards/plane.jpg', 'BACKEND','hi!');
+INSERT INTO notice(name, salary, image, job_position, content) VALUES('bossdog2', 5000, 'https://cdn.vuetifyjs.com/images/cards/road.jpg', 'BACKEND','hi!');
+INSERT INTO notice(name, salary, image, job_position, content) VALUES('bossdog3', 5000, 'test', 'BACKEND','hi!');
+INSERT INTO notice(name, salary, image, job_position, content) VALUES('bossdog4', 5000, 'https://cdn.vuetifyjs.com/images/cards/house.jpg', 'BACKEND','hi!');
+INSERT INTO notice(name, salary, image, job_position, content) VALUES('bossdog5', 5000, 'https://cdn.vuetifyjs.com/images/cards/plane.jpg', 'BACKEND','hi!');
+INSERT INTO notice(name, salary, image, job_position, content) VALUES('bossdog6', 5000, 'https://cdn.vuetifyjs.com/images/cards/road.jpg', 'BACKEND','hi!');
+INSERT INTO notice(name, salary, image, job_position, content) VALUES('bossdog7', 5000, 'test', 'BACKEND','hi!');
+INSERT INTO notice(name, salary, image, job_position, content) VALUES('yeji1', 6000, 'test', 'BACKEND','hi!');
+
+
+INSERT INTO language(notice_id, languages) values (1, 'CPP');
+INSERT INTO language(notice_id, languages) values (1, 'JAVA');
+INSERT INTO language(notice_id, languages) values (2, 'CPP');
+INSERT INTO language(notice_id, languages) values (3, 'CPP');
+INSERT INTO language(notice_id, languages) values (4, 'CPP');
+INSERT INTO language(notice_id, languages) values (5, 'CPP');
+INSERT INTO language(notice_id, languages) values (6, 'CPP');
+INSERT INTO language(notice_id, languages) values (7, 'CPP');
+INSERT INTO language(notice_id, languages) values (8, 'CPP');
```

Comment:
```java
@Configuration
public class DummyDataConfiguration {
    @Configuration
    private class LocalDataApplicationRunner implements ApplicationRunner {

        private final AnswerRepository answerRepository;

        public LocalDataApplicationRunner(AnswerRepository answerRepository) {
            this.answerRepository = answerRepository;
        }

        @Override
        public void run(ApplicationArguments args) {
            Answer ans1 = Answer.builder()
                .userId(1L)
                .questionId(1L)
                .content(AnswerContent.from("Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."))
                .build();
            Answer ans2 = Answer.builder()
                .userId(1L)
                .questionId(1L)
                .content(AnswerContent.from("content2"))
                .build();
            Answer ans3 = Answer.builder()
                .userId(1L)
                .questionId(1L)
                .content(AnswerContent.from("content3"))
                .build();
            Answer ans4 = Answer.builder()
                .userId(1L)
                .questionId(1L)
                .content(AnswerContent.from("content4"))
                .build();
            Answer ans5 = Answer.builder()
                .userId(1L)
                .questionId(1L)
                .content(AnswerContent.from("content5"))
                .build();

            answerRepository.save(ans1);
            answerRepository.save(ans2);
            answerRepository.save(ans3);
            answerRepository.save(ans4);
            answerRepository.save(ans5);
        }
    }
}
```

위와 같이 더미 데이터를 삽입해보는 것은 어떨까요 ?
실제 객체를 만들어서 작성하는 것이므로 SQL에서 발생하는 휴먼 에러를 피할 수 있다고 생각되네요.

Reply:
오 이런 방법도 있었군요...!
이렇게하면 테스트코드에 데이터가 안 올라가나요..?
또 프로덕션 코드에 업로드시 해당 파일들은 gitignore를 올려야겠죠..?